degu_utils.c

```diff
@@ -245,6 +252,16 @@ int degu_get_asset(void)
 		goto a71ch_end;
 	}
 
+	code_delete = degu_coap_request("x509/key", COAP_METHOD_DELETE, NULL, NULL);
+	if (code_delete < COAP_RESPONSE_CODE_OK) {
```

Comment:
Key and Cert should delete at once. I think case of connection lost when middle of Key deleting and Cert deleting that  should not check one by one, delete anyway both of them.

Reply:
OK, I change to send only DELETE "x509/key".
Degu GW delete Key and Cert, when receive DELETE "x509/key".

Reply:
> Degu GW delete Key and Cert, when receive DELETE "x509/key".
Great. That`s neat.